longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
+variable "pipeline_stage" {}
+variable "name" {}
+variable "host" {}
+
+variable "web_size" {
+  default = "Standard-1X"
+}
+
+variable "web_scale" {
+  default = "1"
+}
+
+variable "queue_size" {
+  default = "Standard-1X"
+}
+
+variable "queue_scale" {
+  default = "1"
+}
+
+variable "redis_type" {
+  default = "hobby-dev"
+}
+
+variable "database_name" {}
+
+variable "database_type" {
+  default = "db.t2.medium"
+}
+
+variable "database_size" {
+  default     = 100
+  description = "The amount of storage to allocate to the database, in GB."
+}
+
+variable "email_name" {}
+variable "email_address" {}
+
+variable "papertrail_destination" {}
+
+data "aws_ssm_parameter" "database_username" {
+  name = "/${var.name}/rds/username"
```

Comment:
I don't know enough about this. But will the username always be in rds? if you can see it changing, consider declaring it as a const in the beginning of the file.

Reply:
Yep! This "data resource" loads the configuration variable from [AWS's Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html), so this can basically be considered a constant.